.github/workflows/ci-build-run.yml

```diff
@@ -0,0 +1,60 @@
+name: CI Build and Run
+
+# see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+# Trigger the workflow on push or pull request
+on: [push, pull_request]
+
+jobs:
+  pytest:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, windows-2019]
+
+    # Timeout: https://stackoverflow.com/a/59076067/4521646
+    timeout-minutes: 50
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: prepare build
+      run: |
+        git submodule init
+        git submodule update
+        mkdir bin
+
+    - name: build for Linux
+      if: runner.os == 'linux'
+      run: |
+        cd bin
+        # this will build ITK as HistoReg needs
+        cmake ..
+        # add '-j${N}' to specify number of threads to use for compilation
+        make -j${N}
+        ## wait for ITK to finish building
+        # this is for HistoReg
+        cmake -DCMAKE_INSTALL_PREFIX=$pwd/"ITK-build" ..
+        # add '-j${N}' to specify number of threads to use for compilation
+        make -j${N}
+        # performs installation and executable stripping
+        make install/strip
+
+    - name: build for Windows
+      if: runner.os == 'windows'
+      run: |
+        cd bin
+        # this will build ITK as HistoReg needs
+        cmake ..
+        cmake --build . --target ALL_BUILD
+        ## wait for ITK to finish building
+        # this is for HistoReg
+        cmake -DCMAKE_INSTALL_PREFIX=$pwd\"ITK-build" ..
```

Comment:
This should be `cmake -DITK_DIR=./ITK-build -DCMAKE_INSTALL_PREFIX=./install ..`